runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -483,6 +483,88 @@ bool J9::ValuePropagation::transformIndexOfKnownString(
    return false;
    }
 
+bool J9::ValuePropagation::transformUnsafeCopyMemoryCall(TR::Node *arraycopyNode)
+   {
+   if (!canRunTransformToArrayCopy())
+      return false;
+
+   TR::TreeTop *tt = _curTree;
+   TR::Node *ttNode = tt->getNode();
```

Comment:
These seem to only get used within the `if` block below. Can we move them there to limit the scope of the variables and make the code easier to reason about?

Reply:
Fixed in https://github.com/eclipse/openj9/compare/d8f5f521fc955e5cb1fbfeaa0d418cb475503a1a..ff0b1e1650935286a1a1c68ca3f5b8d8402e489a